README.md

```diff
@@ -5,31 +5,24 @@
 
 # InputStream Adaptive add-on for Kodi
 
-This is an adaptive file addon for kodi's new InputStream Interface.
+This is a [Kodi](https://kodi.tv) input stream add-on wich acts as a demuxer for segmented, multi-bitrate internet streams. The most common streaming protocols such as MPEG-DASH, HLS and Microsoft Smooth Streaming are supported.
 
-- this addon is part of the official kodi repository and part of each kodi installation
-- configure the addon by adding URL prefixes wich are allowed to be played by this addon
-- create a .strm file / or addon with passes an url with sets inputstream.adaptive.manifest_type to either "mpd", "ism" or "hls" and open the strm file in kodi
-- or write an addon wich passes an mpd or ism manifest file to kodi
+The add-on also has support for DRM protected streams, such as Google Widevine, Microsoft PlayReady and others, for some of them are only available on specific operating systems.
 
-##### Examples:
-1.) mpd dash example with one video and one audio stream
-- Force inputstream.mpd using a property in strm file: #KODIPROP:inputstream=inputstream.adaptive
-- Select the tye of the manifest using a property in strm file: #KODIPROP:inputstream.adaptive.manifest_type=mpd
-- URL to paste into strm file: http://download.tsi.telecom-paristech.fr/gpac/DASH_CONFORMANCE/TelecomParisTech/mp4-live/mp4-live-mpd-AV-BS.mpd
+The add-on can be used from all video add-ons by appropriately configuring the *ListItem* properties to instruct Kodi to play the content by using this add-on.
```

Comment:
```suggestion
The add-on can be used from all video add-ons by appropriately configuring the *ListItem* properties to instruct Kodi to play the content via this add-on.
```

Reply:
"Via" is valid.

I think "add-on" is repeated too often, but I'm not sure how to rephrase: _The add-on can be used from all video add-ons by appropriately configuring the *ListItem* properties to instruct Kodi to play the content via this mechanism._, perhaps?

Reply:
i think to change with something like "To enable your video add-on to instruct Kodi to playback by using this InputStream add-on, you have to appropriately configuring the *ListItem* properties." ?